actionview/test/template/form_helper/form_with_test.rb

```diff
@@ -381,6 +381,26 @@ def test_form_with
     assert_dom_equal expected, output_buffer
   end
 
+  def test_form_with_with_persisted_to_model
+    post_form = Struct.new(to_model: @post, keyword_init: true)
```

Comment:
It will not work well.
If it use struct, we should be written as follows.
```suggestion
    post_form = Struct.new(:to_model, keyword_init: true).new(to_model: @post)
```

Reply:
I've rolled back the change to use `Struct`. Thank you!

Reply:
Sorry, @yujideveloper is correct, I meant to suggest `Struct.new(:to_model, keyword_init: true).new(to_model: @post)`. You could also hoise the new struct definition to the top of the test case (eg. `RecordForm = Struct.newStruct.new(:to_model, keyword_init: true)`). Thanks for pointing that out.

It looks like we're using open structs again, so I'll explain a little more why I'm suggesting structs. In short, they are slow. There's even a [cop](https://www.rubydoc.info/gems/rubocop/0.61.1/RuboCop/Cop/Performance/OpenStruct) for this. Yes, this is just tests, but I think it is reasonable to still write faster code where we can.

Reply:
@gmcgibbon good point, a test file-local `RecordForm` makes sense.

Reply:
@gmcgibbon I've force-pushed a version that relies on `Struct`.